### PR TITLE
python sdk release notes updated

### PR DESCRIPTION
**Pull request by jenniferopal**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the CODEOWNERS. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [X] Maintenance/Chore

### CODE REVIEW

File: release-notes/feature-flags.md [hunk 1]
```diff
@@ -57,6 +57,13 @@ Harness deploys changes to Harness SaaS clusters on a progressive basis. This me
  - Add Gradle dependency-check plugin.
  - `ch.qos.logback:logback-classic` has been upgraded from 1.2.11 to 1.3.12. (FFM-10373)
 
+### Python SDK
+
+#### Version 1.4.0
+
+ - You can now add `get_flag_type` public method which enables you to check the type of a flag at any time. If you'd like to see an example of this, have a look at the [`get_flag_type.py`](https://github.com/harness/ff-python-server-sdk/blob/main/examples/get_flag_type_example/get_flag_type.py).(FFM-10393)
```
Comment (erdirowlands) [inline release-notes/feature-flags.md:64]: nitpick
```suggestion
 - You can use the new `get_flag_type` public method to check the type of a flag at any time. If you'd like to see an example of this, have a look at the [`get_flag_type.py`](https://github.com/harness/ff-python-server-sdk/blob/main/examples/get_flag_type_example/get_flag_type.py).(FFM-10393)
```

**Review by erdirowlands — APPROVED** [review]
looks good!  minor nitckpick

File: release-notes/feature-flags.md (hunk 1 above)
Comment (jenniferopal) [reply release-notes/feature-flags.md:64]: Thank you, I'll put that in. :)

**Review by HarnessTheChaos — DISMISSED** [review]
LGTM

**Review by Kira-Jennings — APPROVED** [review]

**Pull request merged**